### PR TITLE
OPSEXP-907 add tagging 'latest' skipci

### PR DESCRIPTION
**Pull request by alxgomz**
add the 'latest' tag to pushed released (follow version tag)

refs OPSEXP-907

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -434,6 +434,11 @@ jobs:
           export GIT_TAG="v$VERSION"
           git tag $GIT_TAG -a -m "Generated tag from TravisCI for build $TRAVIS_BUILD_NUMBER"
           git push https://$GITHUB_TOKEN@github.com/Alfresco/acs-deployment $GIT_TAG
+          git tag -d latest
+          git tag -a -m "current lastest -> $GIT_TAG" -f latest ${GIT_TAG}^{}
```
[the remark this answers is not in the document]
Comment (alxgomz) [reply .travis.yml:438]: OPSEXP-907 really mentions "latest" and not "lastest".
Also according to the link below, latest is what we want (... I think):
https://www.askdifference.com/lastest-vs-latest/

[the remark this answers is not in the document]
Comment (mteodori) [reply .travis.yml:438]: no lastest rather than latest is in the commit message and should be amended to match :)

**Review by mteodori — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (alxgomz) [reply .travis.yml]: Had to read it 3 times before spotting it

File: .travis.yml [hunk 2]
```diff
@@ -434,6 +434,11 @@ jobs:
...
+          for ref in ':refs/tags/latest' 'latest'
```
[the remark this answers is not in the document]
Comment (alxgomz) [reply .travis.yml:439]: Well spotted, changed to the right repo.

**Review by mteodori — APPROVED** [review]

**Pull request merged**